File: PHP/abc018A.php
<?php
$n = [];
$r = [1,1,1];
$i = 0;
while ($line = fgets(STDIN)) {
  $a = $line;
  $m = intval($a);
  $n[$i] = $m;
  $i = $i + 1;
}
for($i = 0; $i < 3; $i++){
  for($j = 0; $j < 3; $j++){
    if($n[$i] < $n[$j]){
      $r[$i] = $r[$i] + 1;
    }
  }
}
print $r[0] ."\n";
print $r[1] ."\n";
print $r[2] ."\n";

File: C/abc018A.c
#include <stdio.h>

int main(void){
  static char line[64];
  int n[3];
  int r[3] = {1,1,1};
  int a, i, j;
  i = 0;
  while (fgets(line, sizeof(line), stdin) != NULL) {
    sscanf(line, "%d", &a);
    n[i] = a;
    i++;
  }
  for (i = 0;i < 3; i++){
    for(j = 0;j < 3; j++){
      if(n[i] < n[j]){
        r[i]++;
      }
    }
  }
  printf("%d\n", r[0]);
  printf("%d\n", r[1]);
  printf("%d\n", r[2]);
}
